history-of-the-quakers-v4/original/14-book-7-chapter-8.adoc

```diff
@@ -1724,8 +1777,10 @@ O grave where is thy victory?
 but thanks be to God who giveth us the victory.`" Many friends
 from London and elsewhere went to visit him,
 which he took very kindly,
-and to one of them he expressed him self in the following manner,
+and to one of them he expressed himself in the following manner,
 
+[quote]
+____
```

Comment:
I think let's switch this one to an embedded testimony.

Reply:
I always forget about that testimony when it's someone speaking!!